code/workspaces/infrastructure-api/src/controllers/stackController.spec.js

```diff
@@ -167,7 +169,7 @@ describe('Stack Controller', () => {
       const response = httpMocks.createResponse();
 
       return stackController.getOneById(request, response)
-        .then(() => {
+        .catch(() => {
           expect(response.statusCode).toBe(500);
           expect(response._getData()).toEqual({ error: 'error', message: 'Error matching ID for stack' }); // eslint-disable-line no-underscore-dangle
         });
```

Comment:
If the request doesn't throw, the test passes without the expects being run, we've been switching to
```
let error;
try {
  await something_that_throws;
} catch (err) {
  error = err;
}
expect(error).toBe...
```

Reply:
Yes, good point. I was just pleased that I spotted this test was nonsense in the first place!

Reply:
updated